### ISSUE TITLE
IDMC Helix - v2.05 production: Proposal: Being able to link master facts to other master facts, or having some sort of entity in between master facts and facts

### ISSUE THREAD

**Issue by MartaLindstrom** [idmc-labs/idmc-database-feedback#288]
Need: When doing analysis of the figures, it sometimes happens that we have a partial (added) figure that is based on multiple facts. We could of course simply link these multiple facts to the master fact (if we get the possibility to flexibly attach sub facts to any master facts), but I am afraid that this will be very confusing as it will not be clear how all the sub facts were estimated.

If you had an entity where you could cluster several facts and do an analysis on a partial figure, which you could then attach to the GRID master fact, I think that the analysis would be easier to understand to the reviewer/D&A staff.

E.g. let's say that I know the number of IDPs in a state in Myanmar as of Jan 2017, but then get an update in August 2017, saying that the number of IDPs have decreased since Jan 2017. However, I get another update in December 2017, saying that the number of IDPs have decreased even more, but this time I only get an update on the population in one of five camps. 

To estimate the outflow from the camps in this state during 2017 I would need to first calculate the number of IDPs in the camps as of December by taking the August figure, subtracting the August estimate for the one camp for which have a more recent estimate, then adding the new figure for this camp as of December. When I thus know the number of IDPs in these camps as of December 2017 I would need to assign this fact with the role of partial (added) and then assign the January total the role partial (subtracted) to get to the estimated returns outflow from the camps.

Note that this calculation only concerns one partial (added) figure that will be needed to be added with multiple other partial (added) figures to get to the final GRID 2018 returns/resettlement figure.

In short, the current system does not accommodate our needs when we need to draw on multiple figures in order to calculate totals that should have the role "partial (added)" or "partial (subtracted)" in the GRID 2018 master facts.

**Comment by MartaLindstrom** [comment]
PS. You can thus have "chains" of facts that lead up to the total estimate, and you may need to use clustering entities (e.g. master facts or similar entities) at several stages in the analysis to produce the final figure.
